From c88526bc2c13793afcc13599ec3b98ca7f77d3f2 Mon Sep 17 00:00:00 2001
From: Etaash Mathamsetty
 <45927311+Etaash-mathamsetty@users.noreply.github.com>
Date: Mon, 13 Jul 2026 15:33:27 -0400
Subject: [PATCH 193/201] winewayland: Dump exstyle in
 wayland_win_data_get_config.

---
 dlls/winewayland.drv/window.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/dlls/winewayland.drv/window.c b/dlls/winewayland.drv/window.c
index 1c8b976b330..a415de1612d 100644
--- a/dlls/winewayland.drv/window.c
+++ b/dlls/winewayland.drv/window.c
@@ -140,14 +140,15 @@ static void wayland_win_data_get_config(struct wayland_win_data *data,
                                         struct wayland_window_config *conf)
 {
     enum wayland_surface_config_state window_state = 0;
-    DWORD style;
+    DWORD style, exstyle;
 
     conf->rect = data->rects.visible;
     conf->window_rect = data->rects.window;
     conf->client_rect = data->rects.client;
     style = NtUserGetWindowLongW(data->hwnd, GWL_STYLE);
+    exstyle = NtUserGetWindowLongW(data->hwnd, GWL_EXSTYLE);
 
-    TRACE("window=%s style=%#x\n", wine_dbgstr_rect(&conf->rect), style);
+    TRACE("window=%s style=%#x exstyle=%#x\n", wine_dbgstr_rect(&conf->rect), style, exstyle);
 
     conf->minimized = FALSE;
 
-- 
2.55.0

